spec/SPEC-annex-C-source-embedding.md

```diff
@@ -0,0 +1,77 @@
+## Annex C
+
+Annex B contains a method of embedding an Input Manifest Identifier into source code files files.
```

Comment:
```suggestion
Annex C contains a method of embedding an Input Manifest Identifier into source code files files.
```

Reply:
Fixed in https://github.com/omnibor/spec/pull/52/commits/115e1111fd41688b6af8ba2855f1184bc1a4c98b